src/main/resources/openapi/transcriptions.yaml

```diff
@@ -0,0 +1,60 @@
+openapi: 3.0.1
+
+servers:
+  - url: http://localhost:4550/
+
+info:
+  description: Modernised DARTS (Digital Audio Recording and Transcription Service).
+  version: "0.0.1"
+  title: Modernised DARTS
+
+paths:
+  /transcriptions:
+    post:
+      tags:
+        - Transcription
+      summary: Users can request transcription
+      operationId: requestTranscription
+      description: Adds a user transcription request for processing
+      responses:
+        '200':
+          description: OK
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/TranscriptionRequestDetails'
+        description: Transcription Request item to add
+components:
+  schemas:
+    TranscriptionRequestDetails:
+      type: object
+      required:
```

Comment:
@jackmaloney I noticed it doesn't say it in the ticket / LLD any longer, but the start and end times I remember thinking they would both be mandatory as it only makes sense to request a transcription if there is audio.
So wouldn't the request have the start and end times of the audio for the hearing if we aren't specifically selecting the times? In the case of selecting events from a court log - I think the idea is to choose the first start time and last end time?

Reply:
The controller verifies the start and end dates for 2 specific types as per the ticket: 

Mandatory fields - 
start_date_time if transcription_type_id is the id of 'Specified times' or 'Court log'
end_date_time if transcription_type_id is the id of 'Specified times' or 'Court log'